docs/codeql/codeql-for-visual-studio-code/working-with-codeql-packs-in-visual-studio-code.rst

```diff
@@ -13,6 +13,12 @@ About CodeQL packs
 ------------------
 CodeQL packs are used to create, share, depend on, and run CodeQL queries and libraries. You can publish your own CodeQL packs and download packs created by others. For more information, see ":ref:`About CodeQL packs <about-codeql-packs>`."
 
+Installing dependencies for CodeQL packs in Visual Studio Code
+--------------------------------------------------------------
+To install dependencies for a CodeQL pack in your Visual Studio Code workspace, run the **CodeQL: Install Pack Dependencies** command from the Command Palette and select the packs you want to install dependencies for.
```

Comment:
In the demo gif, this is called **CodeQL: Install Packs** - I assume that the name changed since you recorded that?

Reply:
Yes. sorry about that - the GIF is slightly outdated! We decided to change the command name to be more explicit 😺